package.json

```
@@ -79,7 +79,7 @@
     "shepherd.js": "^11.2.0",
     "short-and-sweet": "^1.0.4",
     "skipto": "^4.1.7",
-    "tinymce": "^8.1.2",
+    "tinymce": "^8.2.2",
     "tinymce-i18n": "^25.9.22",
```

```suggestion
    "tinymce-i18n": "^25.11.17",
```
As said in my previous review, others should be checked, too.

---

And besides this one there still might be some remaining to be checked.